test/unit/org/apache/cassandra/cql3/MemtableSizeTest.java

```diff
@@ -28,17 +28,20 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
-import org.apache.cassandra.Util;
 import org.apache.cassandra.config.DatabaseDescriptor;
 import org.apache.cassandra.db.ColumnFamilyStore;
 import org.apache.cassandra.db.Keyspace;
 import org.apache.cassandra.db.memtable.Memtable;
 import org.apache.cassandra.utils.FBUtilities;
-import org.apache.cassandra.utils.ObjectSizes;
+import org.github.jamm.MemoryMeter;
 
 @RunWith(Parameterized.class)
 public class MemtableSizeTest extends CQLTester
 {
+    // The meter in ObjectSizes uses omitSharedBufferOverhead which counts off-heap data too
+    static final MemoryMeter meter = new MemoryMeter().ignoreKnownSingletons()
+                                                      .withGuessing(MemoryMeter.Guess.FALLBACK_UNSAFE);
```

Comment:
To see a printout of the usage for each object, add `.enableDebug()` here (most useful with smaller number of partitions).